### PR TITLE
[ZT] Add details to custom cert page

### PR DESCRIPTION
**Pull request by maxvp**
PCX-6917

### CODE REVIEW

File: content/cloudflare-one/policies/filtering/http-policies/_index.md [hunk 1]
```diff
@@ -47,11 +47,11 @@ To use this feature, deploy a [custom root certificate](/cloudflare-one/connecti
...
+| Block        | Display [block page](/cloudflare-one/policies/filtering/configuring-block-page/) as set in Zero Trust.                                                                                                                                                                                                                                                                                                         |
+| Pass through | Bypass insecure connection warnings and seamlessly connect to the upstream. To use this feature, deploy a [custom CA certificate](/cloudflare-one/connections/connect-devices/warp/user-side-certificates/custom-certificate/). For more information on what statuses are bypassed, refer to the [troubleshooting FAQ](/cloudflare-one/faq/teams-troubleshooting/#i-see-error-526-when-browsing-to-a-website). |
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/http-policies/_index.md]:
```suggestion
| Pass through | Bypass insecure connection warnings and seamlessly connect to the upstream. To use this feature, deploy a [custom root certificate](/cloudflare-one/connections/connect-devices/warp/user-side-certificates/custom-certificate/). For more information on what statuses are bypassed, refer to the [troubleshooting FAQ](/cloudflare-one/faq/teams-troubleshooting/#i-see-error-526-when-browsing-to-a-website). |
```

File: content/cloudflare-one/connections/connect-devices/warp/user-side-certificates/custom-certificate.md [hunk 2]
```diff
@@ -14,102 +14,110 @@ Only available on Enterprise plans.
 
 Enterprise customers who do not wish to install the [Cloudflare certificate](/cloudflare-one/connections/connect-devices/warp/user-side-certificates/install-cloudflare-cert/) have the option to upload their own root certificate to Cloudflare. Gateway will use your uploaded certificate to encrypt all sessions between the end user and Gateway, enabling all HTTPS inspection features that previously required the Cloudflare certificate. You can upload multiple certificates to your account, but only one can be active at any given time.
 
+When preparing your certificate and private key for upload, be sure to remove any unwanted characters, such as mismatching subdomains in the certificate's common name.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/user-side-certificates/custom-certificate.md:17]: How does a user know if there is an unwanted character?

**Review by ranbel — APPROVED** [review]

File: content/cloudflare-one/connections/connect-devices/warp/user-side-certificates/custom-certificate.md (hunk 2 above)
Comment (maxvp) [reply content/cloudflare-one/connections/connect-devices/warp/user-side-certificates/custom-certificate.md:17]: This is most likely seen in a mismatch (such as a common name not matching the desired domain), or extra characters (like line breaks) from copy-pasting.

Comment (maxvp) [reply content/cloudflare-one/connections/connect-devices/warp/user-side-certificates/custom-certificate.md:17]: If there are unwanted characters, they'll probably run into an SSL 526 error.

**Pull request merged**